### PR TITLE
Con 1492 organisations & agreements

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs [hunk 1]
```diff
@@ -327,5 +310,12 @@ public async Task<ActionResult> WhenDoYouWantToView(int? choice, string agreemen
                 }
             }
         }
+
+        [HttpGet]
+        [Route("agreements/{accountLegalEntityHashedId}/viewallagreements")]
+        public ActionResult ViewAllAgreements(string hashedAccountId, string accountLegalEntityHashedId)
+        {
+           throw new NotImplementedException();
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs:318]: In this a placeholder for Vas's story?

Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs:318]: If you can call this it shouldn't throw it should put up a message - so this story could be shipped independently

Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: Ok has been reverted to previous functionality

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs [hunk 2]
```diff
@@ -240,63 +240,46 @@ public async Task<ActionResult> GetSignedPdfAgreement(string agreementId, string
...
+            var model = await _orchestrator.GetConfirmRemoveOrganisationViewModel(accountLegalEntityHashedId, hashedAccountId, OwinWrapper.GetClaimValue(ControllerConstants.UserRefClaimKeyName));
+
+            return View(model.Data.CanBeRemoved ? ControllerConstants.ConfirmRemoveOrganisationActionName : ControllerConstants.CannotRemoveOrganisationViewName, model.Data);
         }
 
-        [HttpGet]
-        [Route("agreements/remove/{agreementId}")]
-        public async Task<ActionResult> ConfirmRemoveOrganisation(string agreementId, string hashedAccountId)
+        [HttpPost]
+        [Route("agreements/{accountLegalEntityHashedId}/remove")]
```
Comment (Stephek) [inline src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs:252]: Shouldn't the route be?

accounts/{HashedAccountId}/organisations/{accountLegalEntityHashedId}/agreements/remove ?

Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs:252]: No, potentially it should be in a different controller now as the remove is done by account legal entity and not agreement.

**Pull request merged**